app/controllers/bonus_credits_controller.rb

```diff
@@ -3,17 +3,20 @@ class BonusCreditsController < ApplicationController
   before_action :authenticate_control!
 
   def index
-    @bonus_credits = BonusCredit.where(game: current_game).order(:round, :team_name, :amount)
+    @bonus_credits = BonusCredit.where(game: current_game).order(:round, :team_id, :amount)
   end
 
   def new
     @round = current_game.round
-    @countries = Game::COUNTRIES
     @bonus_credits = BonusCredit.new
+    @teams = Team.all
```

Comment:
Wait, why? Team is agnostic to game. Team is like, "`Brazil`". Even in the event that we're querying off of Team, we should be passing specifics. So, this is used for a dropdown where you select which team someone belongs to - in which case the game is already on the `user` model? I don't mind either way, I'm mostly just uncertain what circumstance would happen where we'd be concerned about that query. There should only ever be ~9 records in the Team table.